### ISSUE TITLE
Font awesome icons need to be marked as hidden for screenreaders

### ISSUE THREAD

**Issue by neiljohari** [Syosset/syosset#90]
We should be using the font-awesome-rails helper method, but it looks like `aria-hidden` isn't even set there. Maybe we could monkey patch it or something.
